edit-post/assets/stylesheets/main.scss

```diff
@@ -13,6 +9,12 @@ body.gutenberg-editor-page {
 		padding-bottom: 0;
 	}
 
+	/* We hide legacy notices in Gutenberg, because they were not designed in a way that scaled well.
+	   Plugins can use Gutenberg notices if they need to pass on information to the user when they are editing. */
+	#wpbody-content > div:not( .gutenberg ):not( #screen-meta ) {
```

Comment:
Don't you think it's a bit too aggressive to hide everything like that? can't we like target notices specifically?

Reply:
Is there a CSS class we could hook into, that every existing plugin utilizes? When I looked at this a while back it seemed like notices were a complete wild west where plugin devs could create their own undismissable and totally custom notices.